### DISCUSSION TITLE
ESP32 S3 with ST7789 170x320 SPI display

### DISCUSSION THREAD

**Discussion by Limme7** [BitMaker-hub/NerdMiner_v2#338] [General]
Hello everyone. I am wondering how to connect my IPS display 1.9inch to my ESP32S3 board in order to display the data like in the lilygo display. As far as i see in the BitMaker code i see that they are using e_SPI lib. But in order to define my pinouts i need to regenerate the bin file and that is a bit too advance for me. So is there any other way and if no can someone direct me from where to start reading and following instructions. Thanks ! :)

**Comment by sany3001** [comment]
> Hello everyone. I am wondering how to connect my IPS display 1.9inch to my ESP32S3 board in order to display the data like in the lilygo display. As far as i see in the BitMaker code i see that they are using e_SPI lib. But in order to define my pinouts i need to regenerate the bin file and that is a bit too advance for me. So is there any other way and if no can someone direct me from where to start reading and following instructions. Thanks ! :)

It shall be easy to build: https://github.com/BitMaker-hub/NerdMiner_v2/issues/306#issuecomment-1959131743

**Reply by djcasper1975** [reply]
did you manage to fix it. i can now build but need to know what to change to fix this.

**Reply by Nimo789** [reply]
I am not the OP, but I stumbled across the same issue: I am trying to connect a small 1.3" OLED IIC Display to my ESP32 WROOM NodeMCU  in order to display the mining stats. As far as I found out by now, the SDA pin of the display should connect to pin G21 of the ESP, while the SCL pin should go to pin G22. GND is connected to Gnd and VCC either to 3V3 or V5. However, I have no clue what I have to modify in the code or the PlatformIO file... I tried to set "-D LED_DISPLAY=1" in the Platformio file, but this only leads to critical errors and the file can not be built... what am I missing here?
